### ISSUE TITLE
Expose method to customize the MetroLog.LogManager.DateTimeFormat 

### ISSUE THREAD

**Issue by ptedeschi** [novotnyllc/MetroLog#77]
The DateTimeFormat  is defined as an internal const.
Developers should be able to change the format string.

**Comment by Aymeeeric** [comment]
I agree with this suggestion ... I will also add customization to the timezone that seems impossible.

**Comment by eresing** [comment]
I also would like to change the time from UTC.

**Comment by Raamakrishnan** [comment]
I used this workaround to show local time using a custom layout.
http://stackoverflow.com/questions/43992460/logging-in-local-time-in-metrolog/43993585#43993585
We can use the default layouts as templates
https://github.com/onovotny/MetroLog/blob/dev/MetroLog.Shared/Layouts/SingleLineLayout.cs
https://github.com/onovotny/MetroLog/blob/dev/MetroLog.Shared/Layouts/FileSnapshotLayout.cs
